organization/_posts/2018-05-02-packaging.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "HSF Packaging Group Meeting #23, May 2, 2018"
+layout: default
+---
+
+# {{page.title}}
+
+Agenda
+======
+[https://indico.cern.ch/event/719557/](https://indico.cern.ch/event/719557/)
+
+Participants: Graeme Stewart, Pere Mato, Guilherme Amadio, Benedikt Hegner, Chris Burr, Ben Morgan, Liz Sexton-Kennedy, Raphal Pacholek, Emil Obreshkov, Ben Couturier, Geri Ganis, Chris Green, Giulio Eulisse, Patricia Mendez, Marco Clemencic, Lynn Garren, Jim Amundson, Patrick Gartung
```

Comment:
There was someone else on Vidyo (Micha Moskovic), but I think it was someone who had left themselves logged in. Unless someone else knows them let's leave them out.